wagtail/documents/templates/wagtaildocs/multiple/add.html

```diff
@@ -29,7 +29,7 @@
                     {% field label_text=label_text id_for_label="id_adddocument_collection" classname="w-mx-auto w-mt-4 w-grid w-justify-center"  %}
                         <select id="id_adddocument_collection" name="collection">
                             {% for pk, display_name in collections.get_indented_choices %}
-                                <option value="{{ pk|unlocalize }}">
+                                <option value="{{ pk|unlocalize }}" {% if pk|unlocalize == selected_collection_id %}selected{% endif %}>
```

Comment:
```suggestion
                                <option value="{{ pk|unlocalize }}"{% if pk|unlocalize == selected_collection_id %} selected{% endif %}>
```

Just a small nitpick in these ones, if you include the 'space' in the if statement we wont get an extra bit of whitespace when the value is not true.